internal/controllers/topology/cluster/cluster_controller.go

```diff
@@ -164,7 +165,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Re
 	if !cluster.ObjectMeta.DeletionTimestamp.IsZero() {
 		// TODO: When external patching is supported, we should handle the deletion
```

Comment:
maybe also adjust the TODO comment?

Reply:
o.O what does that comment even mean? :)

Reply:
I didn't mean your comment, I meant the TODO comment in the code :) 

I just have no idea what that has to do with external patching

Reply:
Ah. There is a 2nd line, that makes more sense

> 		// TODO: When external patching is supported, we should handle the deletion
>		// of those external CRDs we created.

But should be still fine to drop as we decided not to implement external patching by creating additional CRs/CRDs to implement external patching